lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -1103,9 +1104,12 @@ export class UserAgentApplication {
             throw AuthError.createUnexpectedError("Hash was not parsed correctly.");
         }
         if (parameters.hasOwnProperty("state")) {
+            const parsedState = RequestUtils.parseLibraryState(parameters.state);
+
             stateResponse = {
                 requestType: Constants.unknown,
-                state: parameters.state,
+                state: parsedState.state,
+                iat: parsedState.iat,
```

Comment:
what is iat?

Reply:
Issued At, inspired by the `iat` claim in JWTs.

Reply:
I wonder if it is better to not resemble a token claim here to avoid confusion.
This represents a requested at time right?

Reply:
Yeah, thats fair. Could be something like `ts` (probably should be kept short)

Reply:
its encoded so length should be ok right?

Reply:
Base64 encoded length still grows as the string itself grows. Main benefit encoding the special characters and whitespace.

Reply:
+1 makes sense, I prefer explicit naming but if we need to balance it with size thats fine